test/__tests__/Question.test.js

```diff
@@ -50,7 +52,7 @@ describe("Question", () => {
       component.container.getElementsByClassName("show-irrelevant");
 
     expect(hiddenQuestionElement).toBeInTheDocument();
-    expect(irrelevantSectionElement).toHaveLength(1);
+    expect(irrelevantSectionElement.length).toBeGreaterThan(0);
   });
 
   it("renders hidden-question when relevant with regular styling", () => {
```

Comment:
```suggestion
  it("renders hidden question when relevant with regular styling", () => {
```

Reply:
In previous test you had this convention.

Reply:
Not sure what you mean here

Reply:
That in other tests you wrote "hidden question" and now you wrote "hidden-question", so i wanted to unify